models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1591 @@
+   shallow-depth soil variables) require less spinup time.  However, other CLM variables
+   take longer to equilibrate to atmospheric forcing (e.g. biomass, soil carbon).     
+
+
+This initial ensemble spinup was run with resolution ``f09_09_mg17`` (0.9x1.25 grid resolution)
+with compset ``2000_DATM%GSWP3v1_CLM50%BGC-CROP_SICE_SOCN_MOSART_SGLC_SWAV`` (CESM run with 
+only land and river components active).  The starting point of the assimilation is run in
+CLM 'hybrid' mode which allows the starting date of the assimilaton to be different than
+the reference case, and loosens the requirements of the system state.  The tradeoff is that
+restarting in hybrid mode does not provide bit-by-bit reproducible simulations.
+
+For the tutorial, set the ``DART_parms.csh`` variables such that the end of the
+ensemble spinup (at time 1-1-2011) are used as the initial conditions for the assimilation:
+
+::
+
+ setenv refcase      clm5.0.06_f09_80
+ setenv refyear      2011
+ setenv refmon       01
+ setenv refday       01
+ setenv reftod       00000
+ ...
+ ...
+ setenv stagedir /glade/p/cisl/dares/RDA_strawman/CESM_ensembles/CLM/CLM5BGC-Crop/ctsm_${reftimestamp}
+ ...
+ ...
+ setenv start_year    2011
+ setenv start_month   01
+ setenv start_day     01
+ setenv start_tod     00000
+
+
++---------------------------+-------------------------------------------------------------+
+| Important variables       |  Description                                                |
+| to set initial conditions |                                                             |
++===========================+=============================================================+
+| refcase                   | The reference casename from the spinup ensemble that serves |
+|                           | as the starting conditions for the assimilation.            |
++---------------------------+-------------------------------------------------------------+
+| refyear, refmon, refday   | The year, month, day and time of day of the reference case  |
+| reftod                    | that the assimilation will start from.                      |
++---------------------------+-------------------------------------------------------------+
+| stagedir                  | The directory location of the reference case files.         |
++---------------------------+-------------------------------------------------------------+
+
+
+
+Step 7: Setting up the observations to be assimilated 
+-----------------------------------------------------
+
+In 'Step 4: Compiling DART' we have already completed an important
+step by executing ``preprocess`` which generates source code 
+(``obs_def_mod.f90``, ``obs_kind_mod.f90``) that supports the assimilation of observations
+used for this tutorial.  In this step, we link the observations to the DART
+code so they can be read during the assimilation. The observations are read into the
+assimilation through an observation sequence file whose format is described 
```

Comment:
If you can, avoid the work link here.  Link is to something external (e.g. linking to the netcdf library) whereas the obs_def_mod.f90 and obs_kind_mod.f90 are compiled into DART.  I think you want people to get the takaway that if they rerun preprocess they have to _recompile_ filter. 

```suggestion
used for this tutorial.  In this step, we compile these observation definitions in to the DART
executables. The observations are read into the
assimilation through an observation sequence file whose format is described 
```